lib/apiservers/engine/backends/container.go

```diff
@@ -57,6 +57,32 @@ import (
 	"github.com/vmware/vic/pkg/uid"
 )
 
+// valid filters as of docker commit 49bf474
+var acceptedPsFilterTags = map[string]bool{
```

Comment:
There are two maps -- `acceptedPsFilterTags` that is what docker supports per the commit shown.  So they are all true because they are all supported by docker.  I have another that's `unSupportedPsFilters` those are not supported by vic and are all false.  As part of the filter validation we use docker code, so these maps follow their design.